proto/dymensionxyz/dymension/dymns/params.proto

```diff
@@ -0,0 +1,267 @@
+    (gogoproto.nullable) = false
+  ];
+
+  // Price of 4 letters Dym-Name, apply to first year
+  string name_price_4_letters = 4 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"name_price_4_letters\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Price of 5+ letters Dym-Name, apply to first year
+  string name_price_5plus_letters = 5 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"name_price_5plus_letters\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Price of 1 letter Alias, one off payment
+  string alias_price_1_letter = 6 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"alias_price_1_letter\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Price of 2 letters Alias, one off payment
+  string alias_price_2_letters = 7 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"alias_price_2_letters\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Price of 3 letters Alias, one off payment
+  string alias_price_3_letters = 8 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"alias_price_3_letters\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Price of 4 letters Alias, one off payment
+  string alias_price_4_letters = 9 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"alias_price_4_letters\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Price of 5 letters Alias, one off payment
+  string alias_price_5_letters = 10 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"alias_price_5_letters\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Price of 6 letters Alias, one off payment
+  string alias_price_6_letters = 11 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"alias_price_6_letters\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Price of 7+ letters Alias, one off payment
+  string alias_price_7plus_letters = 12 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"alias_price_7plus_letters\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Price extends Dym-Name yearly, after first year
+  string price_extends = 13 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"price_extends\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Denom of the price setup
+  string price_denom = 14 [
+    (gogoproto.moretags) = "yaml:\"price_denom\""
+  ];
+
+  // Minimum price of an offer.
+  // Mostly used to prevent spamming and abusing store with low price offers,
+  // so the value should not be so low.
+  string min_offer_price = 15 [
```

Comment:
buy order can be placed multiple, while sell-order not.
Sell orders placed by the owner, can be one at a time. And well come to spam because Dym-Name, which was bought at higher amount, will be stolen. Anyway, there is a gas mechanism, charges when create Sell-Order so this is no need.